packages/neos-ui-redux-store/src/UI/ContentCanvas/index.js

```diff
@@ -13,6 +13,7 @@ const SET_PREVIEW_URL = '@neos/neos-ui/UI/ContentCanvas/SET_PREVIEW_URL';
 const SET_SRC = '@neos/neos-ui/UI/ContentCanvas/SET_SRC';
 const FORMATTING_UNDER_CURSOR = '@neos/neos-ui/UI/ContentCanvas/FORMATTING_UNDER_CURSOR';
 const SET_CURRENTLY_EDITED_PROPERTY_NAME = '@neos/neos-ui/UI/ContentCanvas/SET_CURRENTLY_EDITED_PROPERTY_NAME';
+const DOCUMENT_INITIALIZING = '@neos/neos-ui/UI/ContentCanvas/DOCUMENT_INITIALIZING';
```

Comment:
@dimaip Regarding the `Saga` aspect: Yeah, that is absolutely correct and I totally agree on the fact that it would just be one line in the `ContentCanvas` reducer. On the other hand: This would lead to breaking the boundaries of the `ContentCanvas` reducer because it would react to an action type which shouldn't be the responsibility of that particular reducer. For me it looks more like a transaction (where the classical saga pattern fits nicely) for "communicating" or "mediate" between contexts.

What is your take on that? :)